march_rqt_gait_generator/test/nosetests/modifiable_subgait_test.py

```diff
@@ -0,0 +1,140 @@
+import copy
+import unittest
+
+from mock import Mock
+import rospkg
+from urdf_parser_py import urdf
+
+from march_rqt_gait_generator.model.modifiable_setpoint import ModifiableSetpoint
+from march_rqt_gait_generator.model.modifiable_subgait import ModifiableSubgait
+
+
+class ModifiableSubgaitTest(unittest.TestCase):
+    def setUp(self):
+        self.gait_generator = Mock()
+        self.gait_name = 'walk'
+        self.subgait_name = 'left_swing'
+        self.version = 'MV_walk_leftswing_v2'
+        self.resources_folder = rospkg.RosPack().get_path('march_rqt_gait_generator') + '/test/resources'
+        self.robot = urdf.Robot.from_xml_file(self.resources_folder + '/march4.urdf')
```

Comment:
You can use the `march4.urdf` in the `march_description` package for this :wink: